src/js/themes/dark.js

```diff
@@ -66,7 +66,25 @@ export default deepFreeze({
     },
   },
   icon: {
-    color: textColor,
+    extend: css`${(props) => {
+      let { color } = props.theme.icon;
+      if (props.color && props.color !== 'plain') {
+        color = colorForName(props.color, props.theme);
+      }
+      if (props.dark === colorIsDark(color)) {
+        color = props.dark ?
+          props.theme.global.colors.darkBackground.text :
+          props.theme.global.colors.lightBackground.text;
+      }
+      return `
+        fill: ${color};
+        stroke: ${color};
+      `;
+    }
+} `,
+  },
+  anchor: {
```

Comment:
why was this added?

Reply:
Not sure if you mean the anchor settings, but dark color Anchors are invisible when on brand background

Reply:
yes, I do:
default theme (watch the anchor and select colors): https://grommet-nextjs.herokuapp.com/?theme=grommet 
dark theme (watch the anchor and select colors): https://grommet-nextjs.herokuapp.com/?theme=dark